sheet_test.go

```diff
@@ -463,3 +465,30 @@ func TestAttrValToFloat(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, 42.1, got)
 }
+
+func TestSetSheetBackgroundFromBytes(t *testing.T) {
+	files := map[string]string{
+		"excelize.svg":          ".svg",
+		"test/images/excel.emf": ".emf",
```

Comment:
We need to consider the Windows directory separator in the path, I suggest using the `filepath.Join` to connect the directory name and file name.